src/v3/src/transformer/i18n/transform.ts

```diff
@@ -73,6 +83,65 @@ const updateLabel = (transaction: IdxTransaction, element: FieldElement): void =
   }
 };
 
+const transformCustomizedErrorKeys = (
+  transaction: IdxTransaction,
+  formBag: FormBag,
+  widgetProps: WidgetProps,
+): void => {
+  const { nextStep: { name } = {} } = transaction;
+  if (name !== IDX_STEP.IDENTIFY) {
+    return;
+  }
+
+  const { dataSchema, uischema: { elements } } = formBag;
+  [
+    { field: 'identifier', key: 'error.username.required' },
+    { field: 'credentials.passcode', key: 'error.password.required' },
+  ].forEach((obj) => {
+    const ele = getUIElementWithName(
+      obj.field,
+      elements as UISchemaElement[],
+    ) as FieldElement;
+    if (ele && isCustomizedI18nKey(obj.key, widgetProps)) {
+      // eslint-disable-next-line no-param-reassign
+      dataSchema[obj.field] = {
```

Comment:
i18n override should not touch validation logic, try find an approach to consume the overriding translation in validate function directly